script/Common/00_00.txt

```diff
@@ -248,7 +248,7 @@ All except one thing.
 [sha:36585715a143e0af2fca7f6b8f857ddab79116c2]{
 -- Page 17, Offset 156.
 -- 涙するような<約束|ことば>を、何か大切なことを忘れている気がして、窓の外をながめ続けた。
-I continued staring through the window, unable to shake off the feeling that I had forgotten something very important. A 'promise' that was bringing me to tears.
+I continued staring through the window, unable to shake off the feeling that I had forgotten something very important. A %{i}promise%{/i} that was bringing me to tears.
```

Comment:
Not sure this needs any special treatment at all tbh, it's rubied but the ruby is just 'word'.